src/main/kotlin/com/ritense/iko/connectors/bag/BagConfig.kt

```diff
@@ -0,0 +1,72 @@
+package com.ritense.iko.connectors.bag
+
+import com.ritense.iko.connectors.bag.endpoints.BagEndpointAdresseerbareObjecten
+import com.ritense.iko.connectors.bag.endpoints.BagEndpointAdressen
+import com.ritense.iko.connectors.bag.endpoints.BagEndpointBronhouders
+import com.ritense.iko.connectors.bag.endpoints.BagEndpointLigplaatsen
+import com.ritense.iko.connectors.bag.endpoints.BagEndpointNummeraanduidingen
+import com.ritense.iko.connectors.bag.endpoints.BagEndpointOpenbareRuimten
+import com.ritense.iko.connectors.bag.endpoints.BagEndpointPanden
+import com.ritense.iko.connectors.bag.endpoints.BagEndpointStandplaatsen
+import com.ritense.iko.connectors.bag.endpoints.BagEndpointVerblijfsobjecten
+import com.ritense.iko.connectors.bag.endpoints.BagEndpointWoonplaatsen
+import org.apache.camel.CamelContext
+import org.apache.camel.component.rest.openapi.RestOpenApiComponent
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+@ConditionalOnProperty(
+    value = ["iko.sources.bag.enabled"],
+    havingValue = "true",
+    matchIfMissing = true
+)
+class BagConfig {
+
+    @Bean
+    fun bag(camelContext: CamelContext, @Value("\${iko.sources.bag.host}") host: String) =

```

Comment:
spring property contains `sources`, should this (and others) be changed as well?